src/wx/panel.cpp

```diff
@@ -1476,7 +1476,7 @@ class FilterThread : public wxThread {
     // if NULL, end thread
     uint8_t* src;
 
-    ExitCode Entry()
+    int Entry()
```

Comment:
Seems that it wants `void` . Should I change it to that and remove change all the returns to `return;`?